src/pages/public_registration/LogPanel.tsx

```diff
@@ -120,20 +126,58 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
               </Typography>
```

Comment:
kanskje denne også skulle tatt hele bredden nå som det er støtte for det?
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/1207f2b3-d7bc-4512-829e-170d23634b4d)

Reply:
Ja, det er en god idé. Per nå er hele denne teksten i første celle, men kan flytte institusjon og bruker ned en rad 👍